contracts/PrizeFlush.sol

```diff
@@ -52,11 +56,10 @@ contract PrizeFlush is IPrizeFlush, Manageable {
         IStrategy _strategy,
         IReserve _reserve
     ) Ownable(_owner) {
-        destination = _destination;
-        strategy = _strategy;
-        reserve = _reserve;
+        _setDestination(_destination);
+        _setReserve(_reserve);
+        _setStrategy(_strategy);
 
-        // Emit Deploy State
         emit Deployed(_destination, _reserve, _strategy);
```

Comment:
General question... do we need a Deployed event if all the setters also have an event?

Seems not necessary in that circumstance.

Reply:
Events are only emitted in the public functions. I think it's best to keep a Deployed event like in other contracts.